new_platforms/libstdio/enc/optee/files_optee.c

```diff
@@ -28,6 +30,29 @@ typedef struct _OPTEE_FILE {
 
 int errno = 0;
 
+/* OP-TEE secure storage (RPMB) is not quite a filesystem: there are no
+ * directories and there are no file names. Instead, one stores objects, each
+ * of which is identified by an at most 64-byte long byte sequence. This byte
+ * sequence need not be a sensible string; it is merely treated as a (void *).
+ * Seeing as oe_fopen(...) is used with file paths and that these can be longer
+ * than 64 characters (i.e. bytes), we don't pass the file path down to OP-TEE.
+ * Instead, we hash it first using SHA-512, which conveniently generates a
+ * 64-byte long digest and use that as the object ID.
+ * 
+ * NOTE: This breaks file enumeration.
+ */
+static void sha512( const unsigned char *input, size_t ilen,
```

Comment:
Fix whitespace (no space after open paren, no space before close paren), for consistency with rest of file.  Same in 49-53

Reply:
Argh, every time I use Visual Studio this happens and sometimes I just miss it. There is a `.clang-format` file at the root of the repo that VS picks up on. It uses `clang`'s formatting utility according to the rules in that file and it wrecks havoc all over.